.github/stale.yml

```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
```

Comment:
I don't think that right now we need to automatically determines if a PR or Issue is stale by inactivity, right now we have just a few issues/PR and right now I think we even have some issues that are running for than 90 days

Reply:
I agree, but I don't see as a problem to already have this part automated. It's more like a quality of life, so it's a nice to have.